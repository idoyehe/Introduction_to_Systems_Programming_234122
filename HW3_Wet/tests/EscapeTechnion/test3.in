# Michal & Amit test 1

company add rnd@apple.com 13
company add rnd@google.com 5
company add rnd@qualcomm.com 3
company add rnd@microsoft.com 13
company add rnd@refael.com 2
# random comment
room add rnd@apple.com 1 100 3 06-18 1
room add rnd@google.com 2 120 4 10-23 3
room add rnd@google.com 1 240 6 13-19 10
room add rnd@microsoft.com 3 160 2 11-14 5
room add rnd@apple.com 2 200 4 09-20 7

escaper add esc1@g.com 13 4
escaper add esc2@g.com 13 6
escaper add esc3@g.com 13 10
escaper add esc4@g.com 13 6
escaper add esc5@g.com 1 8
escaper add esc6@g.com 5 8
escaper add esc7@g.com 6 7
escaper add esc8@g.com 7 4

escaper order esc1@g.com 13 1 0-10 4
escaper order esc7@g.com 5 1 0-13 3
escaper recommend esc6@g.com 5
escaper order esc2@g.com 13 3 2-13 3
escaper order esc2@g.com 13 1 1-10 3

report best

report day
escaper order esc8@g.com 13 2 0-17 4
report best
escaper recommend esc3@g.com 3
report day
report day
report best

escaper remove esc1@g.com
escaper remove esc2@g.com
escaper remove esc3@g.com
escaper remove esc4@g.com
escaper remove esc5@g.com
escaper remove esc6@g.com
escaper remove esc7@g.com
escaper remove esc8@g.com

room remove 13 1
room remove 13 2
room remove 13 3
room remove 5 1
room remove 5 2

company remove rnd@apple.com
company remove rnd@google.com
company remove rnd@qualcomm.com
company remove rnd@microsoft.com
company remove rnd@refael.com