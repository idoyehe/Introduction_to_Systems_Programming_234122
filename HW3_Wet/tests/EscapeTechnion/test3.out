The system has 18 faculties and in 0 days, they have made a revenue of 0
One of the best faculties - Civil and Environmental Engineering, has made a revenue of 0 so far!
One of the best faculties - Mechanical Engineering, has made a revenue of 0 so far!
One of the best faculties - Electrical Engineering, has made a revenue of 0 so far!

Day #0: 3 events in total
The order was made by esc6@g.com, from the faculty of Aerospace Engineering, who is skilled at 8
	It was made for room 2, of rnd@apple.com from Computer Science, for the time 9:00
	The room is best for a skill skill_level of 7. 5 people came for the total price of 1000
The order was made by esc1@g.com, from the faculty of Computer Science, who is skilled at 4
	It was made for room 1, of rnd@apple.com from Computer Science, for the time 10:00
	The room is best for a skill skill_level of 1. 4 people came for the total price of 300
The order was made by esc7@g.com, from the faculty of Industrial Engineering and Management, who is skilled at 7
	It was made for room 1, of rnd@google.com from Aerospace Engineering, for the time 13:00
	The room is best for a skill skill_level of 10. 3 people came for the total price of 720
That is all for day #0!

The system has 18 faculties and in 1 days, they have made a revenue of 2020
One of the best faculties - Computer Science, has made a revenue of 1300 so far!
One of the best faculties - Aerospace Engineering, has made a revenue of 720 so far!
One of the best faculties - Civil and Environmental Engineering, has made a revenue of 0 so far!

Day #1: 3 events in total
The order was made by esc2@g.com, from the faculty of Computer Science, who is skilled at 6
	It was made for room 1, of rnd@apple.com from Computer Science, for the time 10:00
	The room is best for a skill skill_level of 1. 3 people came for the total price of 225
The order was made by esc3@g.com, from the faculty of Computer Science, who is skilled at 10
	It was made for room 1, of rnd@google.com from Aerospace Engineering, for the time 13:00
	The room is best for a skill skill_level of 10. 3 people came for the total price of 720
The order was made by esc8@g.com, from the faculty of Mathematics, who is skilled at 4
	It was made for room 2, of rnd@apple.com from Computer Science, for the time 17:00
	The room is best for a skill skill_level of 7. 4 people came for the total price of 800
That is all for day #1!

Day #2: 1 events in total
The order was made by esc2@g.com, from the faculty of Computer Science, who is skilled at 6
	It was made for room 3, of rnd@microsoft.com from Computer Science, for the time 13:00
	The room is best for a skill skill_level of 5. 3 people came for the total price of 360
That is all for day #2!

The system has 18 faculties and in 3 days, they have made a revenue of 4125
One of the best faculties - Computer Science, has made a revenue of 2685 so far!
One of the best faculties - Aerospace Engineering, has made a revenue of 1440 so far!
One of the best faculties - Civil and Environmental Engineering, has made a revenue of 0 so far!

